website/content/en/v0.20.0/getting-started/migrating-from-cas/scripts/step11-create-provisioner.sh

```diff
@@ -5,12 +5,12 @@ metadata:
   name: default
 spec:
   requirements:
-    - key: karpenter.k8s.aws/instance-family
+    - key: karpenter.k8s.aws/instance-category
       operator: In
       values: [c, m, r]
     - key: karpenter.k8s.aws/instance-generation
       operator: Gt
-      values: ["2"]
+      values: ["4"]
```

Comment:
this excludes `G3`

Reply:
Good catch, I think we can leave the instance generation as-is. Anyways, we are planning to migrate away from all of these different getting-started guides with #3017

Reply:
@jonathan-innis  I also had left comments on that PR too about the same thing :)